app/src/main/java/edu/mit/mitmobile2/maps/MapsActivity.java

```diff
@@ -1,13 +1,7 @@
 package edu.mit.mitmobile2.maps;
```

Comment:
This file is unnecessary, all of the maps functionality is implemented in the MapsFragment